runtime/Makefile

```diff
@@ -15,7 +15,7 @@ ENABLE_LIGHTNING_KOKKOS?=ON
 ENABLE_OPENQASM?=ON
 ENABLE_ASAN?=OFF
 # TODO: Update to v0.35.0 after the next lightning release.
-LIGHTNING_GIT_TAG_VALUE?="2716864521c539429bd382b92151a918d1a076ce"
+LIGHTNING_GIT_TAG_VALUE?="efb946a736f824835ebf4fd692568d49e6cf1ca5"
```

Comment:
There should be more locations than this to update, double check all our actions to make sure :)

Reply:
Good catch!

Reply:
https://github.com/PennyLaneAI/catalyst/pull/558/commits/10ff1ecee8305b4c32b81d3788247f61fbc81128